cli-wrap/src/main/resources/cli/da-cli.py

```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/env python3
 import sys
-from listings import *
 import argparse
+import inspect
+import os
+
+cmd_folder = os.path.realpath(os.path.abspath(os.path.split(inspect.getfile( inspect.currentframe() ))[0]))
+if cmd_folder not in sys.path:
+     sys.path.insert(0, cmd_folder)
+
```

Comment:
Why are you inserting the path to sys.path? can't it be done just by prepending the path to config file name?